best_practices/01_reference_guide.md

```diff
@@ -0,0 +1,14 @@
+## Naming Conventions
```

Comment:
Good to have this, but these are not our current naming conventions, they are however outlined in the contributing document

Reply:
Overall it's unclear who the audience is for this document.